cmd/mt-whisper-importer-writer/main.go

```diff
@@ -0,0 +1,283 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"math"
+	"net/http"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/dgryski/go-tsz"
+	"github.com/gocql/gocql"
+	"github.com/kisielk/whisper-go/whisper"
+	"github.com/raintank/dur"
+	"github.com/raintank/metrictank/api"
+	"github.com/raintank/metrictank/cluster"
+	"github.com/raintank/metrictank/cluster/partitioner"
+	"github.com/raintank/metrictank/idx"
+	"github.com/raintank/metrictank/idx/cassandra"
+	"github.com/raintank/metrictank/mdata"
+	"github.com/raintank/metrictank/mdata/chunk"
+	"github.com/raintank/metrictank/mdata/chunk/archive"
+)
+
+var (
+	exitOnError = flag.Bool(
+		"exit-on-error",
+		true,
+		"Exit with a message when there's an error",
+	)
+	verbose = flag.Bool(
+		"verbose",
+		false,
+		"Write logs to terminal",
+	)
+	fakeAvgAggregates = flag.Bool(
+		"fake-avg-aggregates",
+		true,
+		"Generate sum/cnt series out of avg series to accomodate metrictank",
+	)
+	httpEndpoint = flag.String(
+		"http-endpoint",
+		"127.0.0.1:8080",
+		"The http endpoint to listen on",
+	)
+	cassandraAddrs = flag.String(
+		"cassandra-addrs",
+		"localhost",
+		"cassandra host (may be given multiple times as comma-separated list)",
+	)
+	cassandraKeyspace = flag.String(
+		"cassandra-keyspace",
+		"metrictank",
+		"cassandra keyspace to use for storing the metric data table",
+	)
+	cassAddr = flag.String(
+		"cass-addr",
+		"localhost",
+		"Address of cassandra host.",
+	)
+	ttlsStr = flag.String(
+		"ttls",
+		"35d",
+		"list of ttl strings used by MT separated by ':'",
```

Comment:
why `:` ? don't we use `,` to separate values everywhere else? (in "multi-level" things such as aggSettings we use `:` but that doesn't apply here)